### ISSUE TITLE
Seconds to HH:MM:SS Bug

### ISSUE THREAD

**Issue by sedj601** [HanSolo/toolbox#4]
Bug in package eu.hansolo.toolbox.Helper

`public static final long[] secondsToHHMMSS(final long seconds)`

should be 

```
public static final long[] secondsToHHMMSS(final long seconds) {
        long secs    = seconds % 60;
        long minutes = (seconds / 60) % 60;
        long hours   = (seconds / (60 * 60)) % 24;
        return new long[] { hours, minutes, secs };
 }
```

I would also rename `secondsToHHMMString `to `secondsToHHMMSSString`

**Comment by HanSolo** [comment]
That’s funny, I‘ve stumbled upon this issue 3 days ago in one of my Swift apps and fixed it there. Thy for the heads up, will fix it.👍🏻

**Comment by HanSolo** [comment]
Fixed with release 21.0.17
